docs/developers/contributing/testing.md

```diff
@@ -191,62 +191,62 @@ See also [this paper on property-based testing in science](https://conference.sc
 ### Testing with `Qt` and `napari.Viewer`
 
 There are a couple things to keep in mind when writing a test where a `Qt` event
-loop or a `napari.Viewer` is required.  The important thing is that any widgets
-you create during testing need to be cleaned up at the end of each test:
+loop or a {class}`~napari.Viewer` is required.  The important thing is that any widgets
+you create during testing need to be cleaned up at the end of each test. We thus
+recommend that you use the following fixtures when needing a widget or
+{class}`~napari.Viewer` in a test.
 
-1. If you need a `QApplication` to be running for your test, you can use the
-   [`qtbot`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot) fixture from `pytest-qt`
+#### qtbot
 
-    > note: fixtures in pytest can be a little mysterious, since it's not always
-    > clear where they are coming from.  In this case, using a pytest-qt fixture
-    > looks like this:
+If you need a `QApplication` to be running for your test, you can use the
+[`qtbot`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot)
+fixture from `pytest-qt`, a napari testing dependency.
+
+```{note}
+Fixtures in pytest can be a little mysterious, since it's not always
+clear where they are coming from.  In this case using the `pytest-qt` `qtbot`fixture
+looks like this:
 
     ```python
```

Comment:
don't think the indentation is needed if the admonition has 4 backticks.

Reply:
Interesting, I'll give it a go. (note I haven't set up building the docs locally :grimacing:, I noped out at the 3 repos part)

Reply:
I don't mind the indentation though, visually nicer? Can change though

Reply:
Hmm, looks like the indents need to go?
<img width="896" alt="image" src="https://github.com/napari/docs/assets/76622105/7641c932-e184-42b8-ae24-d3b0387345f0">
Happy to be corrected, but I have no other explanation anyways.